src/lib.rs

```diff
@@ -113,6 +113,10 @@
 //!
 //! Please read [the migration guide](https://github.com/embedded-graphics/embedded-graphics/blob/master/MIGRATING-0.5-0.6.md).
 //!
+//! # Migrating from 0.6 to 0.7
+//!
+//! Please read [the migration guide](https://github.com/embedded-graphics/embedded-graphics/blob/master/MIGRATING-0.6-0.7.md).
+//!
```

Comment:
We could merge both migration guide links into one section.

```suggestion
//! # Migrating from older version
//!
//! * [Migration guide from 0.5 to 0.6](https://github.com/embedded-graphics/embedded-graphics/blob/master/MIGRATING-0.5-0.6.md).
//! * [Migration guide from 0.6 to 0.7](https://github.com/embedded-graphics/embedded-graphics/blob/master/MIGRATING-0.5-0.6.md).
//!
```